Pod/Classes/BONChain.h

```diff
@@ -119,14 +119,16 @@ typedef BONChain *BONCNonnull (^BONTagComplexStyles)(BONGeneric(NSArray, BONTag
 
 // concatenation
 - (void)appendLink:(id<BONTextable>)link;
-- (void)appendLink:(id<BONTextable>)link separator:(BONNullable NSString *)separator;
+- (void)appendLink:(id<BONTextable>)link separatorTextable:(BONNullable id<BONTextable>)separator;
 
 @end
 
 @interface BONChain (Deprecated)
 
 @property (copy, nonatomic, readonly) BONChainColor textColor __attribute__((deprecated("use -color instead")));
 
+- (void)appendLink:(id<BONTextable>)link separator:(BONNullable NSString *)separator __attribute__((deprecated("use -appendLink:link separatorTextable:separator instead")));
```

Comment:
#req the way to spell the name of the new method is `-appendLink:separatorTextable:`.